pom.xml

```diff
@@ -119,6 +119,21 @@
             <artifactId>commons-collections</artifactId>
             <version>3.2.2</version>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
```

Comment:
what is Jackson? Is there an alternative?

Reply:
Jackson is a data processing tool, but famous with generating and parsing JSON. In short, it is a library used to convert java object into json format. Alternative libraries are: 
JSON.simple, GSON, JSONP etc...

Reply:
will you please list all the alternative libraries and add to serializer class as private classes like we did for jackson and gson